tests/test_reduce_ops.py

```diff
@@ -0,0 +1,74 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import numpy as np
+from op_test import OpTest
+
+
+class TestReduceSumOp(OpTest):
+    def setUp(self):
+        self.init_op_type()
+        self.init_keep_dim()
+        self.init_reduce_all()
+        self.inputs = {'X': np.random.random((5, 6, 7, 8)).astype("float32")}
+        self.attrs = {
+            'dim': 2,
+            'keep_dim': self.keep_dim,
+            'reduce_all': self.reduce_all
+        }
+        self.outputs = {'Out': np.zeros((1, 1))}
+
+    def init_op_type(self):
+        self.op_type = "reduce_sum"
```

Comment:
Small thing, but just be consistent whether you use single or double quotes. Small formatting thing